contracts/base/ExecutionHelper.sol

```diff
@@ -140,8 +140,10 @@ contract ExecutionHelper is IExecutionHelperEventsAndErrors {
     function _handleSingleExecution(bytes calldata executionCalldata, ExecType execType) internal {
         (address target, uint256 value, bytes calldata callData) = executionCalldata.decodeSingle();
         if (execType == EXECTYPE_DEFAULT) _execute(target, value, callData);
-        else if (execType == EXECTYPE_TRY) _tryExecute(target, value, callData);
-        else revert UnsupportedExecType(execType);
+        else if (execType == EXECTYPE_TRY) {
+            (bool success, bytes memory result) = _tryExecute(target, value, callData);
+            if (!success) emit TryExecuteUnsuccessful(0, result);
```

Comment:
I had also gotten feedback that the information we emit for these internal success or failure got to make more sense. for example it should emit calldata and not emit things like indexes 

I will share the finding description

Reply:
https://github.com/bcnmy/nexus/pull/135 @livingrockrises

Reply:
why there are two prs with same work?